bossdata/spec.py

```diff
@@ -16,6 +16,36 @@
 import astropy.table
 
 
+def get_fiducial_pixel_index(wavelength):
+        """
+        Convert a wavelength to a fiducial pixel index.
+
+        Args:
+            wavelength(float): Input wavelength in Angstroms.
+
+        Returns:
+            numpy.ndarray: Array of floating-point indices relative to the fiducial
+                wavelength grid.
+        """
+        return (np.log10(wavelength) - _fiducial_log10lam0)/_fiducial_coef
```

Comment:
Typically, lya bao analyses, neighboring pixels are combined in groups of 3. A general purpose version of this function is useful for determining indices on the combined wavelength grid (w/ `coef = _fiducial_coef*num_combine`).